pyproject.toml

```diff
@@ -81,7 +81,7 @@ docs = [
     "sphinx-lint==0.9.1",
 ]
 apt = [
-    "python-apt>=2.4.0;sys_platform=='linux'"
+    "python-apt @ https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/python-apt/2.4.0ubuntu2/python-apt_2.4.0ubuntu2.tar.xz ; sys_platform == 'linux'"
```

Comment:
```suggestion
        "python-apt>=2.4.0;sys_platform=='linux'",
```

Reply:
Reasoning: A developer should be matching the version of `python-apt` that their machine comes with.

Reply:
Oops, yeah I changed this for my local environment and accidentally committed it.

Why do we prefer short form?  It doesn't work on my (24.04) system.